ci/recipe/meta.yaml

```diff
@@ -13,11 +13,12 @@ build:
 
 requirements:
   build:
-    - python {{ python }}
     - setuptools
```

Comment:
I think setuptools goes in the host section:
https://conda.io/docs/user-guide/tutorials/build-pkgs.html#editing-the-meta-yaml-file

So build is strictly for compilers now (and hopefully still npm for webpack).